src/hooks/useGame.hook.js

```diff
@@ -21,17 +21,18 @@ export default function useGame(marksToWin = 3) {
       return
     }
 
-    if (marksInBoard === NUMBER_OF_POSITIONS) {
-      alert('DRAW GAME!')
-      handleOnReset()
-      return
-    }
-
     const theWinner = getWinner(board)
 
     if (theWinner) {
       setWinner(theWinner)
       setNextPlayer(NO_PLAYER)
+      return
+    }
+
+    if (marksInBoard === NUMBER_OF_POSITIONS) {
+      alert('DRAW GAME!')
+      handleOnReset()
+      return
```

Comment:
In this case it is not necessary to add this return because there is no more code below